sync-core/src/main/java/com/cloudant/http/HttpConnectionManager.java

```diff
@@ -0,0 +1,47 @@
+package com.cloudant.http;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+/**
+ * Created by tomblench on 23/03/15.
+ */
+public class HttpConnectionManager {
+
```

Comment:
Inclined to agree this class could just be noise -- did it have a greater role to play before or have a planned greater role?